### PR TITLE
(Issue #1566) UI Fix -Create a Casebook Pages

### PR DESCRIPTION
**Pull request by shljessie**
### Add Content Button UI Fix Suggestions

1. (title) Add Resource --> Add Content 
> consistency with button name 

2. Tab Names 

> Find a Legal Document --> Find a Caselaw
add descriptions to what is in the search database
Catherine: it contains us code sections & caselaws

> Create Custom Content
add descriptions on what this could be. 
Can it be a section for discussion questions? Series of related photos? Related Essays? 
Catherine: intros to a case, questions following, videos 

> Add Links
add descriptions on the difference btw adding a link(image) here and adding a link to the create a custom content page.

### CODE REVIEW

File: web/frontend/components/AddContent.vue [hunk 1]
```diff
@@ -5,20 +5,23 @@
...
     >Add Content</button>
     <Modal v-if="showModal" @close="showModal = false" :initial-focus="focusTarget">
-      <template slot="title">Add Resource</template>
+      <template slot="title" title="Add Content">Add Resource</template>
       <template slot="body">
         <div class="search-tabs">
           <a
             v-bind:class="{ active: caseTab, 'search-tab': true }"
             v-on:click.stop.prevent="setTab('case')"
+            title="Find a Caselaw"
```
Comment (lizadaly) [inline web/frontend/components/AddContent.vue:14]:
```suggestion
            title="Find Caselaw"
```

File: web/frontend/components/AddContent.vue [hunk 2]
```diff
@@ -5,20 +5,23 @@
...
           >Find a Legal Document</a>
           <a
             v-bind:class="{ active: textTab, 'search-tab': true }"
             v-on:click.stop.prevent="setTab('text')"
+            title="Create a Custom Content"
```
Comment (lizadaly) [inline web/frontend/components/AddContent.vue:19]:
```suggestion
            title="Create some Custom Content"
```
I think we could have a more descriptive label overall but let's defer thinking about that for now.

**Review by lizadaly — APPROVED** [review]
Added some suggested changes for wording but otherwise this looks good! Love the comments for places to improve later.

**Comment by rebeccacremona** [comment]
@shljessie says this can be closed

**Pull request closed without merging**